scripts/build-home.js

```diff
@@ -16,14 +18,15 @@ async function cleanBuildDir() {
 
 async function build() {
   await cleanBuildDir()
-  let bundler = new Parcel({
-    entries: join(SRC, 'index.pug'),
-    defaultConfig: join(ROOT, 'node_modules', '@parcel', 'config-default'),
-    patchConsole: false,
-    sourceMaps: false,
-    mode: 'production'
-  })
-  await bundler.run()
+  await viteBuild({
```

Comment:
Since we have vite.config.js we don't need to have inline config here

Reply:
i tried, but unfortunately we need to have inline config here, because build function not use vite.config.js(

Reply:
Hmm, it should work if add all options to vite.config.js

May need to use [conditional config](https://vitejs.dev/config/#conditional-config) because there two entrypoints (one for api and another for home page)

Reply:
its work!)